hack/testing/rsyslog/roles/rsyslog/tasks/main.yaml

```diff
@@ -0,0 +1,159 @@
+---
+- name: Install required packages
```

Comment:
This installs the packages and not affected by the "use_rsyslog_image" parameter. Should the when condition be:
when: rsyslog__enabled|bool and not use_rsyslog_image|bool  ?